modules/custom/social_advanced_queue/social_advanced_queue.module

```diff
@@ -7,16 +7,30 @@
 
 use Drupal\social_user\Plugin\Action\SocialSendEmail;
 use Drupal\social_advanced_queue\Plugin\Action\SocialSendEmailAdvancedQueue;
+use Drupal\social_advanced_queue\Plugin\ActivitySend\EmailActivitySendAdvancedQueue;
 
 /**
  * Implements hook_action_info_alter().
  */
 function social_advanced_queue_action_info_alter(&$definitions) {
   // The Send Email Action needs a new Queue Worker.
-  // So we can extend it.
+  // So we override it and create Advanced Queue Jobs instead of Queue Items.
```

Comment:
Shall we also introduce deprecation notices for this queue **SocialSendEmailAdvancedQueue** class, if we are planning to move to advance_queues in OS 11.x?

Reply:
It's indeed added to: https://www.drupal.org/project/social/issues/3215750#comment-14116094
So we can deprecate it once that starts :D